src/status.cc

```diff
@@ -176,5 +176,161 @@ std::string getStatusString(Status status) {
   return "";
 }
 
+
+std::string getStatusName(Status status) {
```

Comment:
Probably better to fill in the response code details in the upstream filter.

Reply:
I was kind of debating.

Prefer to put this util function in this lib as it looks like a common usage like `getStatusString`.